COCOAPODS.md

```diff
@@ -1,150 +1,381 @@
+            
+            pod("AFNetworking", "~> 4.0.0")
+            pod("SDWebImage/MapKit")
+        }
+    }
+    ```
+    
+    </div>
 
-To import a Kotlin/Native module in an existing Xcode project:
+2. Re-import the project.
 
-0. Make sure that you have CocoaPods [installed](https://guides.cocoapods.org/using/getting-started.html#installation).
-We recommend using CocoaPods 1.6.1 or later.
+If you want to use dependencies on local pods from Kotlin code, import the corresponding packages.
 
-1. Configure a Gradle project: apply the `org.jetbrains.kotlin.native.cocoapods` plugin, add
-and configure the targets, and specify the required podspec fields.
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
 
-2. Run the `podspec` task. The podspec file described above will be generated.
+```kotlin
+import cocoapods.pod_dependency.*
+import cocoapods.subspec_dependency.*
+```
 
-    > In order to avoid compatibility issues during an Xcode build, the plugin requires using
-    a [Gradle wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html). 
-    To generate the wrapper automatically during execution of the `podspec` task,
-    run it with the parameter `-Pkotlin.native.cocoapods.generate.wrapper=true`.
+</div>
 
-3. Add a reference to the generated podspec in a Podfile of the Xcode project.
+You can find a sample project [here](https://github.com/Kotlin/kotlin-with-cocoapods-sample).
+ 
+## Use a Kotlin Gradle project as a CocoaPods dependency
 
-    <div class="sample" markdown="1" theme="idea" mode="ruby">
+You can use a multiplatform project with native targets as a CocoaPods dependency (Kotlin Pod). You can include such a dependency
+  in a `Podfile` of an Xcode project  by its name and path to the project directory containing the generated `Podspec`. 
+This dependency will be automatically built (and rebuilt) along with this project. 
+Such approach simplifies importing to Xcode by removing a need to write corresponding Gradle tasks and Xcode build steps manually.
+ 
+You can add dependencies between:
+* [A Kotlin Pod and an Xcode project with one target](#add-a-dependency-between-a-kotlin-pod-and-xcode-project-with-one-target)
+* [A Kotlin Pod and an Xcode project with several targets](#add-a-dependency-between-a-kotlin-pod-with-an-xcode-project-with-several-targets)
+* [Several Kotlin Pods and an Xcode project](#add-dependencies-between-multiple-kotlin-pods-and-an-xcode-project)
+
+> To correctly import the dependencies into the Kotlin/Native module, the
+`Podfile` must contain either [`use_modular_headers!`](https://guides.cocoapods.org/syntax/podfile.html#use_modular_headers_bang)
+or [`use_frameworks!`](https://guides.cocoapods.org/syntax/podfile.html#use_frameworks_bang)
+directive.
+{:.note}
+
+### Add a dependency between a Kotlin Pod and Xcode project with one target
+
+1. Create an Xcode project with `Podfile` if you haven’t done this yet.
```

Comment:
1. Create an Xcode project with a `Podfile` if you haven’t done so yet.

Reply:
✅